media-gfx/{apngdis,farbfeld,icon-slicer}: EAPI8 bump

mm1ke opened the pull request:
Hi,

another few simple `EAPI8` bump. Also removed the live ebuild from `media-gfx/farbfeld`

orbea commented:
> Also removed the live ebuild from `media-gfx/farbfeld`

Just a suggestion, but it would be nice to document why in the commit message.

mm1ke commented:
> > Also removed the live ebuild from `media-gfx/farbfeld`
> 
> Just a suggestion, but it would be nice to document why in the commit message.

Sure, i've updated the commit. Basically the reason is because the last upstream commit was from 2018 and there is only one more commit from the last version which simple updates the copyright header.

ConiKost commented on media-gfx/icon-slicer/icon-slicer-0.3.ebuild:
```diff
@@ -8,7 +8,7 @@ HOMEPAGE="https://www.freedesktop.org/software/icon-slicer/"
-LICENSE="GPL-2"
+LICENSE="MIT"
 SLOT="0"
 IUSE=""
```
If empty, drop it :-)

mm1ke replied:
Thanks, missed that and fixed now
